schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id integer,
```

Comment:
## [OPTIONAL]

- [ ] Nice job :clap:  on  creating the animal table however I would suggest making the id column not null and autoincremented in this table like this :point_down: 

```sql
id INT GENERATED ALWAYS AS IDENTITY NOT NULL,
```